### PR TITLE
core concepts

### PR DESCRIPTION
**Pull request by rkostik**
core concepts -HW

### CODE REVIEW

File: src/components/Button/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './Button.css';
+
+const Button = ()=>(
+        <div className="div-btn">
+        </div>
```
Comment (artem-blazhko) [inline src/components/Button/Button.jsx]: If it is empty you should not close it. 
It should look like this <div className="" />

Comment (artem-blazhko) [reply src/components/Button/Button.jsx]: Also, use appropriate classNames. It is not a good idea to call it like div, span, input, etc.

File: src/components/Class_example/Class_example.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../Func_example/Func_example.css';
+
+class Class_example extends React.Component {
+    render() {
+        return (
+            <div>
+                <input className="main" placeholder={this.props.title}>
+                </input>
```
Comment (artem-blazhko) [inline src/components/Class_example/Class_example.jsx]: You do not need this close element. 
https://react-cn.github.io/react/tips/self-closing-tag.html

File: src/components/Func_example/Func_example.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
+
+.main {
+    position: absolute;
+    width: 713px;
+ height: 57px;
```
Comment (artem-blazhko) [inline src/components/Func_example/Func_example.css]: use appropriate code style. I suppose that there should be space at the beginning

**Comment by rkostik** [comment]
close

**Pull request closed without merging**